aten/src/ATen/native/quantized/cpu/qrelu.cpp

```diff
@@ -134,6 +135,64 @@ Tensor& leaky_relu_quantized_cpu_(Tensor& self, const Scalar& negval) {
   return self;
 }
 
+Tensor prelu_quantized_cpu_impl(const Tensor& self, const Tensor& weight,
+                                double output_scale, int64_t output_zero_point) {
+  // General idea:
+  // For input ndim in [1, 3], create fake dimensions and use NHWC fast path
+  // For input ndim in [4, 5], use NHWC fast path
+  // In other cases, use reference path
+
+  auto ndim = self.dim();
+  // for ndim < 1 or > 5, go to reference path
+  if (ndim > 5 || ndim < 1) {
+    auto x = self.dequantize();
+    auto y = at::prelu(x, weight);
+    return at::quantize_per_tensor(y, output_scale, output_zero_point, c10::kQUInt8);
+  }
+
+  // Create fake dimensions if input ndim in [1, 3]
+  Tensor qx = self;
```

Comment:
we don't need to create a Channels last memory view upon qx here.

TensorIterator will re-calculate the shape of input tensor according to order of strides (re-arrange the dimension order from strides lowest to highest). AND memory format is only a tag informing the strides, so it makes no difference TensorIterator how to deal with it. 

only need to create a view upon `weight` so as to make sure that it's size is able to be broadcasted to input's size.